web/templates/navbar_icon_and_title.hbs

```diff
@@ -1,3 +1,6 @@
+{{#if recent_view}}
+Recent in
```

Comment:
Presumably this needs to be marked as a translation string?

Reply:
Ah yes thanks

Reply:
This also is an incomplete sentence; marking those alone doesn't work given that other languages have different word orders.

Reply:
@timabbott what would be the best way to do this? I see ways to insert variables, but here I'm also inserting an icon in the middle of the sentence.

Reply:
`git grep z-stream-name` for many examples of this syntax.

Reply:
Awesome, thanks!